src/components/PictureGallery/PictureGalleryUtils.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Get a Picture array - first one of each date.
+ * @param {Picture[]} pictures - An array of picture objects.
+ * @returns {Picture[]} - one for of each date
```

Comment:
What's the `Picture` means here? I know JSDoc understands array as a type, how will it understand `Picture`?

Reply:
To tell the truth, I don't really use JSDoc, and in my work, I use typescript so I thought I'll use a typescript like type. but tell me what's more clear to write.

Reply:
We'll start using TypeScript soon, but for now let's stick with the JSDoc spec, which requires to enter just the type.
```suggestion
 * @returns {array} - Array of pictures, one for of each date
```